docs-source/content/release-notes.md

```diff
@@ -8,6 +8,7 @@ draft: false
 
 | Date | Version | Introduces backward incompatibilities | Change |
 | --- | --- | --- | --- |
+| June 20, 2019 | v2.2.1 | no  | The operator now supports Kubernetes 1.14.0+.  This release is primarily a bug fix release and resolves the following issues:<br><ul><li>Servers part of domains on persistent volume would sometimes fail to start during the introspection phase after a full domain shutdown.  The introspection script now better handles the relevant error conditions.</li><li>The domain resource provides an option to <a href="https://github.com/oracle/weblogic-kubernetes-operator/blob/master/docs/domains/Domain.md#server-service">pre-create Kubernetes services</a> for WebLogic servers that are not yet running so that the DNS addresses of these services are resolvable.  These services are now created as non-headless so that they have an IP address.</li></ul> 
```

Comment:
Servers part of domains on persistent volume -> (not exactly sure what you wanted to say here, but ...) Servers in domains on persistent volumes
The introspection script now better handles -> Now, the introspection script better handles
WebLogic servers -> WebLogic Servers